tests/bluetooth/tester/src/mesh.c

```diff
@@ -13,6 +13,8 @@
 #include <misc/byteorder.h>
 #include "bttester.h"
 
+#include "../../../../subsys/bluetooth/host/testing.h"
```

Comment:
Shouldn't this header file be in include/bluetooth?

Reply:
I have mixed feelings adding this to the public headers. Since it will be only used for testing purposes I don't think that this should be exposed. IMO we should prevent using this.

Reply:
The app writer has to knowingly enable CONFIG_BT_TESTING before using the APIs in this header, so I'm not that worried about misuse of it. You should of course clearly document these APIs as well, i.e. state that they're purely for testing (that said, the bt_test_* prefix makes it kind of obvious)

Reply:
done